### PR TITLE
refactor: bridge ad delegation

### PR DESCRIPTION
**Pull request by minh-rakuten**
# Description
Break down `MiniAppMessageBridge`. Move ad implementation into `AdBridgeDispatcher`.
Exclude non-test files in codecov.

# Checklist
- [x] I have read the [contributing guidelines](../CONTRIBUTING.md).
- [x] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `./gradlew check` without errors

### CODE REVIEW

File: miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/AdBridgeDispatcher.kt [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+import com.rakuten.tech.mobile.miniapp.ads.MiniAppAdDisplayer
+import com.rakuten.tech.mobile.miniapp.ads.isAdMobProvided
+
+@Suppress("TooGenericExceptionCaught", "LongMethod")
+internal class AdBridgeDispatcher {
+    private lateinit var bridgeExecutor: MiniAppBridgeExecutor
+    private lateinit var adDisplayer: MiniAppAdDisplayer
+    private var isAdMobEnabled = false
+
+    fun setAdMobDisplayer(adDisplayer: MiniAppAdDisplayer) {
```
Comment (khairul-alam-licon) [inline miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/AdBridgeDispatcher.kt:13]: It means, we need to update the related change in [USERGUIDE](https://github.com/rakutentech/android-miniapp/blob/master/miniapp/USERGUIDE.md#admob)?

Comment (minh-rakuten) [reply miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/AdBridgeDispatcher.kt:13]: No, there is nothing change in user implementation.

**Review by khairul-alam-licon — APPROVED** [review]
Looks good 👍 
Thank you for the refactoring.

**Review by corycaywood — APPROVED** [review]
Looks good 👍

**Pull request merged**